hosting/setup-guides/linux.md

```diff
@@ -86,6 +134,103 @@ Recovery Phrase: poet never rifle awake lunar during ocean eight dial gospel cra
 Address addr:333d10486632f11c4c5b907c2e45d31478522dec525649712697404b4253e92ea5a84227187d
 ```
 
+## Configure Storage Drives
```

Comment:
This section is outside the scope of the guide. You can highlight that drives should be mounted on boot, but lets leave *how* up to the user.